src/components/FiatValue.svelte

```diff
@@ -33,7 +34,7 @@
   $: id = CoinGeckoId[coin.assetHash];
   $: currency = CoinGeckoId[fiat];
   $: promise = getValue(id, amount, currency);
-  $: visible = amount && id;
+  $: visible = $marinaStore.network === NetworkNames.MAINNET && amount && id;
```

Comment:
it seems to me that network can be a prop of the FiatValue component like amount, coin etc.. and passed from the top, instead of requiring access to the store ?

Reply:
Commit added with that change.